Khan/wonder-blocks
# Add Icon Button to Storybook

## Summary:

Add IconButton to storybook. This uses the documentation in styleguidist
as the foundation.

Issue: https://khanacademy.atlassian.net/browse/WB-1138

<img width="1332" alt="Screen Shot 2022-06-29 at 1 57 46 PM" src="https://user-images.githubusercontent.com/13231763/176545779-c1145bfa-82d2-420c-8803-705e1d90a88a.png">
<img width="1327" alt="Screen Shot 2022-06-29 at 1 57 55 PM" src="https://user-images.githubusercontent.com/13231763/176545786-c5ed0c6e-f6b5-4776-9756-3ed2f4b5692b.png">
<img width="1288" alt="Screen Shot 2022-06-29 at 1 58 02 PM" src="https://user-images.githubusercontent.com/13231763/176545794-085053c7-6d55-4b71-8872-7ad7d3a290fa.png">
<img width="1292" alt="Screen Shot 2022-06-29 at 1 58 11 PM" src="https://user-images.githubusercontent.com/13231763/176545801-7ac4b6a5-dbbf-4e13-be1a-5879862e5602.png">
<img width="1298" alt="Screen Shot 2022-06-29 at 1 58 21 PM" src="https://user-images.githubusercontent.com/13231763/176545806-a3630f70-0c5d-44e9-a1c6-564cc640a00c.png">
<img width="1321" alt="Screen Shot 2022-06-29 at 1 58 28 PM" src="https://user-images.githubusercontent.com/13231763/176545814-237a7d27-cea9-4947-a2ab-d9b8e1a931a7.png">


## Test plan:
- Check the descriptions of the stories and make sure they make sense.
- Confirm there are no TODOs in the prop descriptions.
- Go through the stories with a screen reader and make sure the
  aria labels make sense.
